app/javascript/components/maps/main-map/components/recent-imagery/components/recent-imagery-settings/recent-imagery-settings-styles.scss

```diff
@@ -103,6 +103,21 @@
     }
   }
 
+  .refresh .message {
```

Comment:
The only improvement I can think of would be to make this a ui component as you have the same styles and button in two places?